appkeeper/assets/configuration/spec.yaml

```
@@ -0,0 +1,22 @@
+name: AppKeeper
+files:
+- name: appkeeper.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: account
+      required: true
+      description: Your AWS accocunt
+      value:
+        type: string
+        example: "000000000000"
+    - name: integrationToken
+      required: true
+      description: Token for AppKeeper Integration
+      value:
+        type: string
+        example: eyXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX..
+  - template: instances/default
```

```suggestion
  - template: instances/http
  - template: instances/default
```

The HTTP wrapper also comes with a config template for other supported options.